### ISSUE TITLE
🐛 Bug: TODO on many blog image alt texts

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [JoshuaKGoldberg/dot-com#110]
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/joshuakgoldberg-dot-com-next/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

All images should have informative alt text (or `""` if  they're decorative). Accessibility!

### Actual

Oops, I left in `alt: TODO` in some places...
